src/pages/superadmin/tableComponent/__tests__/TableComponent.spec.tsx

```diff
@@ -423,4 +423,64 @@ describe('MyTable Component', () => {
     const paidBounties = getAllByText('paid');
     expect(paidBounties.length).toBe(1);
   });
+
+  it('simulates filtering bounties by status: open, assigned, paid', async () => {
```

Comment:
Also the status calls and sent to the backend

Reply:
same with the other comment is that the API call that sorts by status is done in the parent component but we can mock the `onChangeStatus` call but I think we can write another test for that